$MirOS: ports/www/firesomething/patches/patch-config_autoconf_mk_in,v 1.2 2006/05/11 20:55:42 bsiegert Exp $
$OpenBSD: patch-config_autoconf_mk_in,v 1.1.1.1 2004/02/15 19:37:25 wilfried Exp $
--- config/autoconf.mk.in.orig	Thu Sep 14 18:06:40 2006
+++ config/autoconf.mk.in	Tue Oct 24 14:40:36 2006
@@ -57,13 +57,13 @@ MOZ_PKG_SPECIAL = @MOZ_PKG_SPECIAL@
 prefix		= @prefix@
 exec_prefix	= @exec_prefix@
 bindir		= @bindir@
-includedir	= @includedir@/$(MOZ_APP_NAME)-$(MOZ_APP_VERSION)
+includedir	= $(prefix)/firesomething/include
 libdir		= @libdir@
 datadir		= @datadir@
 mandir		= @mandir@
 idldir		= @datadir@/idl/$(MOZ_APP_NAME)-$(MOZ_APP_VERSION)
 
-mozappdir	= $(libdir)/$(MOZ_APP_NAME)-$(MOZ_APP_VERSION)
+mozappdir	= $(prefix)/firesomething
 mredir		= $(libdir)/mre/mre-$(MOZ_APP_VERSION)
 mrelibdir	= $(mredir)/lib
 
@@ -252,7 +252,7 @@ OS_LDFLAGS	= @LDFLAGS@
 OS_COMPILE_CFLAGS = $(OS_CPPFLAGS) @COMPILE_CFLAGS@
 OS_COMPILE_CXXFLAGS = $(OS_CPPFLAGS) @COMPILE_CXXFLAGS@
 
-OS_INCLUDES	= $(NSPR_CFLAGS) $(JPEG_CFLAGS) $(PNG_CFLAGS) $(ZLIB_CFLAGS)
+OS_INCLUDES	= $(NSPR_CFLAGS) $(FT2_CFLAGS) $(LIBIDL_CFLAGS) $(JPEG_CFLAGS) $(PNG_CFLAGS) $(ZLIB_CFLAGS)
 OS_LIBS		= @LIBS@
 ACDEFINES	= @MOZ_DEFINES@
 
